pkg/shared/shared.go

```diff
@@ -47,3 +48,21 @@ func fileExists(p string) (bool, error) {
 	}
 	return false, fmt.Errorf("checking existance of file %s: %w", p, err)
 }
+
+// ServerAuthSecret returns the secret of the manage server using the secret
+// file as given in environment variable. In case of development it uses the
+// development password.
+func ServerAuthSecret(secretFile string, devEnv string) ([]byte, error) {
+	pw := []byte(DevelopmentPassword)
```

Comment:
Are you sure the constant `DevelopmentPassword` has to be public? Maybe rename it to `developmentPassword`